core/class.go

```diff
@@ -56,7 +56,28 @@ type Cairo1Class struct {
 	Program         []*felt.Felt
 	ProgramHash     *felt.Felt
 	SemanticVersion string
-	Compiled        json.RawMessage
+	Compiled        CompiledClass
+}
+
+type CompiledClass struct {
+	Bytecode        []*felt.Felt        `json:"bytecode"`
+	PythonicHints   json.RawMessage     `json:"pythonic_hints"`
+	CompilerVersion string              `json:"compiler_version"`
+	Hints           json.RawMessage     `json:"hints"`
+	EntryPoints     CompiledEntryPoints `json:"entry_points_by_type"`
+	Prime           string              `json:"prime"`
+}
+
+type CompiledEntryPoints struct {
+	External    []CompiledEntryPoint `json:"EXTERNAL"`
+	L1Handler   []CompiledEntryPoint `json:"L1_HANDLER"`
+	Constructor []CompiledEntryPoint `json:"CONSTRUCTOR"`
```

Comment:
Can there be more than one constructor?

Reply:
For now no, but I'm not sure where I read that in the future it may be. However, I followed the [implementation](https://github.com/xJonathanLEI/starknet-rs/blob/e9bdd7efeae9c76ef62a130ccef3f4355a864797/starknet-core/src/types/contract/mod.rs#L83)

Reply:
Let's go with that.